network/transport/v2/senders.go

```diff
@@ -91,7 +91,6 @@ func (p *protocol) sendTransactionListQuery(id transport.PeerID, refs []hash.SHA
 			Debug("Did not request a TransactionList while another conversation is in progress")
 		return nil
 	}
-	conversation.set("refs", refs)
```

Comment:
We do not check these anymore?

Reply:
We do, but directly on the refs in the TransactionListQuery message that created conversation. The additional properties were unused/unchecked.